components/tabs/tabs.js

```diff
@@ -798,6 +806,14 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 				selectedTab.selected = true;
 				selectedTab.tabIndex = 0;
 			}
+
+			this._tabs.forEach((tab) => {
```

Comment:
Is the panel guaranteed to be in the DOM when this runs? Mostly thinking about the scenario where a consumer might create and append a new tab, and then create and append the associated new panel. Maybe it is a safe assumption. 

If not, we might need to move this into a helper that is also invoked from the `panels` `slotchange` as well, and debounce it so it only runs once. Debouncing it would defer the wire-up by at least a task, so I would be inclined to only do it if necessary.